api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/PersonNameTranslatorImplTest.java

```diff
@@ -163,44 +155,6 @@ public void shouldOnlyCreateOneExtensionForExtensibleAttributes() {
 		assertThat(extension.getExtension().size(), greaterThan(1));
 	}
 	
-	@Test
```

Comment:
So, I ended up getting rid of this test entirely, since it isn't actually testing anything happening intentionally in the FHIR2 module, but is just a symptom of what the OpenMRS `getFullName` method returns.  This allowed me to not have to worry about mocking out the hacky static stuff in NameSupport